nuxeo-features/rest-api/nuxeo-rest-api-test/src/main/java/org/nuxeo/ecm/restapi/test/BaseTest.java

```diff
@@ -155,16 +155,25 @@ protected CloseableClientResponse getResponse(RequestType requestType, String pa
     protected CloseableClientResponse getResponse(RequestType requestType, String path, String data,
             MultivaluedMap<String, String> queryParams, MultiPart mp, Map<String, String> headers,
             String... acceptedTypes) {
+        return getResponse(requestType, path, data, queryParams, mp, "dublincore", headers, acceptedTypes);
+    }
+
+    protected CloseableClientResponse getResponse(RequestType requestType, String path, String data,
+            MultivaluedMap<String, String> queryParams, MultiPart mp, String schemas, Map<String, String> headers,
```

Comment:
Shouldn't `schemas` be passed within the headers?

Reply:
This code is now gone